lib/geordi/commands/deploy.rb

```diff
@@ -94,6 +94,8 @@ def deploy(target_stage = nil)
     git_call << "git merge #{source_branch}" if merge_needed
     git_call << 'git push' if push_needed
 
+    invoke_cmd 'bundle_install'
+
```

Comment:
This change should not be necessary, as there is no Bundler invocation between L98 and L107. The Capistrano call is executed in L108

Reply:
There is a call to `Util.gem_major_version` in L98/L100 which uses `bundle list`

Reply:
:detective: ok!